### PR TITLE
Redesign course page

### LINKED ISSUE #406: Redesign: Course page
This task is to implement the redesign for our course show page as shown in the mock up images below. For reference this is the current course show page for the rails course http://www.theodinproject.com/courses/ruby-on-rails

### Working on this task

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] Check out our [styleguide](https://staging-odin-project.herokuapp.com/styleguide)
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch  

If you have any questions or need clarification about anything related to this task please contact the maintainers in our [contributing channel on Gitter](https://gitter.im/TheOdinProject/Contributing)

### Redesigned Course show page mock up
lesson

### PR DESCRIPTION
**Pull request by arun1595**
Fixes #406

### CODE REVIEW

File: app/assets/stylesheets/global.scss [hunk 1]
```diff
@@ -27,6 +27,10 @@ h2, h3 {
   color: $text-primary;
 }
 
+a, a:hover, a:focus {
+  text-decoration: none;
```
Comment (KevinMulhern) [inline app/assets/stylesheets/global.scss]: Nice! 👍 👍 👍

File: app/views/courses/course/_banner.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div class="row">
+  <div class="col course-banner text-center">
+    <%= image_tag course.badge,
```
Comment (KevinMulhern) [inline app/views/courses/course/_banner.html.erb]: Can we format this a little differently please
```
<%= image_tag course.badge,
  class: 'img-reponsive course-badge',
  alt: "#{course.title.downcase} badge"
%>
```

File: app/views/courses/course/_lessons.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="lessons-list">
+  <% lessons.each_with_index do |lesson, index| %>
+    <%= render 'courses/course/lesson',
```
Comment (KevinMulhern) [inline app/views/courses/course/_lessons.html.erb]: ditto

File: app/views/courses/course/_section.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  <div class="section-title">
+    <h2><%= section.title %></h2>
+  </div>
+
+  <div class="section-description">
+    <p><%= section.description %></p>
+  </div>
+
+  <% lessons = section.lessons.includes(:course).order(:position) %>
+  <%= render 'courses/course/lessons',
```
Comment (KevinMulhern) [inline app/views/courses/course/_section.html.erb]: ditto

File: app/controllers/courses_controller.rb [hunk 5]
```diff
@@ -1,9 +1,16 @@
 class CoursesController < ApplicationController
   def index
-    @courses = Course.order(position: :asc)
```
Comment (KevinMulhern) [inline app/controllers/courses_controller.rb]: Do the specs need to be updated for this?

File: app/views/courses/course/_banner.html.erb (hunk 2 above)
Comment (arun1595) [inline app/views/courses/course/_banner.html.erb]: Sure. Let me do it.

File: app/controllers/courses_controller.rb (hunk 5 above)
Comment (arun1595) [inline app/controllers/courses_controller.rb]: No

File: app/views/courses/course/_section.html.erb (hunk 4 above)
Comment (arun1595) [inline app/views/courses/course/_section.html.erb]: Done.

File: app/views/courses/course/_lessons.html.erb (hunk 3 above)
Comment (arun1595) [inline app/views/courses/course/_lessons.html.erb]: Done.

File: app/views/courses/course/_banner.html.erb (hunk 2 above)
Comment (arun1595) [inline app/views/courses/course/_banner.html.erb]: Done.

**Comment by codyloyd** [comment]
let me pull this one and take a look locally.  Code looks fine on first glance 👍

**Comment by KevinMulhern** [comment]
Thanks @arun1595 
👍 LGTM

**Comment by KevinMulhern** [comment]
@codyloyd is it ready to :shipit: ?

**Comment by codyloyd** [comment]
sure.

**Pull request merged**